packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -415,8 +417,13 @@ export const DetailsTile = ({
                 {isScheduledRelease && track?.release_date ? (
                   <View style={styles.releaseContainer}>
                     <IconCalendarMonth color='accent' size='m' />
-                    <HarmonyText color='accent' strength='strong' size='m'>
-                      Releases on
+                    <HarmonyText
```

Comment:
`import { Text } from '@audius/harmony-native'` is a thing now I think, but ok to keep using this for now